docs/en/deploy/Running-Alluxio-On-a-HA-Cluster.md

```diff
@@ -220,34 +219,31 @@ $ ./bin/alluxio runTests
 ## Access an Alluxio Cluster with HA
 
 When an application interacts with Alluxio in HA mode, the client must know about
-the Alluxio HA cluster, so that the client knows how to discover the Alluxio leading master.
-There are two ways to specify the HA Alluxio service address on the client side:
+the connection information of Alluxio HA cluster, so that the client knows how to discover the Alluxio leading master.
+The following sections listed out two ways to specify the HA Alluxio service address on the client side.
 
 ### Specify Alluxio Service in Configuration Parameters
 
-Users can also pre-configure the service address of an Alluxio HA cluster in environment variables
+Users can pre-configure the service address of an Alluxio HA cluster in environment variables
 or site properties, and then connect to the service using Alluxio URI like to
-`alluxio:///path` where the required connection details to the HA cluster information is already
-configured with these parameters.
-For example, if using Hadoop, you can configure the properties in `core-site.xml`, and then use the
-Hadoop CLI with an Alluxio URI.
+`alluxio:///path` where the required connection details to the HA cluster information is already provided.
+For example, With Alluxio connection information in `core-site.xml` of Hadoop, Hadoop CLI can 
+connect to Alluxio cluster with HA.
 
 ```console
 $ hadoop fs -ls alluxio:///directory
 ```
 
-Depending on the different approach to achieve HA, different properties are required to set:
+Depending on the different approaches to achieve HA, different properties are required:
+
+- `alluxio.master.rpc.addresses` is required when connecting to HA cluster on embedded journal.
```

Comment:
want to make it clear that the two approaches are:
- embedded journal
- zookeeper

so i would start the first bullet point with:
`- If using embedded journal, set ...`
and then start the second with:
`- If using Zookeeper, set...`

Reply:
that's much better